app/__init__.py

```diff
@@ -16,6 +16,19 @@
 s3 = FlaskS3()
 
 
+opensearch_field_name_map = {
+    "file_name": "File name",
+    "description": "Description",
+    "Source-Organisation": "Transferring body",
```

Comment:
does this show up multiple times because we have transferring body name as a field?